kedro-datasets/tests/video/test_video_dataset.py

```diff
@@ -139,21 +122,26 @@ def test_load_missing_file(self, empty_dataset_mp4):
         with pytest.raises(DatasetError, match=pattern):
             empty_dataset_mp4.load()
 
-    def test_save_s3(self, mp4_object, mocked_s3_bucket, tmp_path):
+    def test_save_s3(
+        self, mp4_object, mocked_s3_bucket, mock_fs_args, credentials, tmp_path
+    ):
         """Test to save a VideoDataset to S3 storage"""
         video_name = "video.mp4"
 
         dataset = VideoDataset(
-            filepath=S3_FULL_PATH + video_name, credentials=AWS_CREDENTIALS
+            filepath=S3_FULL_PATH + video_name,
+            fs_args=mock_fs_args,
+            credentials=credentials,
         )
         dataset.save(mp4_object)
 
+        body_stream = mocked_s3_bucket.get_object(
+            Bucket=S3_BUCKET_NAME, Key=f"{S3_KEY_PATH}/{video_name}"
+        )["Body"].read()
         tmp_file = tmp_path / video_name
-        mocked_s3_bucket.download_file(
-            Bucket=S3_BUCKET_NAME,
-            Key=S3_KEY_PATH + "/" + video_name,
-            Filename=str(tmp_file),
-        )
+        with open(tmp_file, "wb") as file:
```

Comment:
nit: we could do `tmp_file.open("wb")` since it's already a pathlib object

Reply:
I didn't knew that was possible :-). Do you want me to change it?